pkg/volume/cinder/cinder.go

```diff
@@ -89,7 +89,7 @@ func (plugin *cinderPlugin) newBuilderInternal(spec *volume.Spec, podUID types.U
 		},
 		fsType:             fsType,
 		readOnly:           readOnly,
-		blockDeviceMounter: &cinderSafeFormatAndMount{mounter, exec.New()}}, nil
+		blockDeviceMounter: &mount.SafeFormatAndMount{plugin.host.GetMounter(), exec.New()}}, nil
```

Comment:
> Fixed. 

Great.. thanks

>(Still, AWS has the same plugin.host.GetMounter() call)

Ah! True...  That is because AWS and Cinder tests do not have this check:
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/gce_pd/gce_pd_test.go#L169 :(